src/kaltura-player.js

```diff
@@ -898,6 +898,9 @@ class KalturaPlayer extends FakeEventTarget {
     const middlewares = [];
     const uiComponents = [];
     const plugins = [];
```

Comment:
I would split into 2 default config files (ovp, ott) as we've talked.

Reply:
done thanks